src/matrix2d.rs

```diff
@@ -401,4 +417,12 @@ mod test {
         assert_eq!(size_of::<Matrix2D<f32>>(), 6*size_of::<f32>());
         assert_eq!(size_of::<Matrix2D<f64>>(), 6*size_of::<f64>());
     }
-}
\ No newline at end of file
+
+    #[test]
+    pub fn test_is_identity() {
+        let m1 = Matrix2D::identity();
+        assert!(m1.is_identity());
```

Comment:
I think the test submodule is allowed to access private functions? If I move the exact same test into another module it fails as expected.